cmssw-tool-conf.spec

```diff
@@ -18,6 +18,7 @@ Requires: starlight-toolfile
 Requires: alpgen-toolfile
 Requires: boost-toolfile
 Requires: bz2lib-toolfile
+Requires: cepgen
```

Comment:
@forthommel , after adding `cepgen-toolfile.spec` please change this to `cepgen-toolfile`